addons/im_livechat/__manifest__.py

```diff
@@ -60,6 +60,8 @@
         'web.qunit_suite_tests': [
             'im_livechat/static/tests/qunit_suite_tests/components/**/*.js',
             'im_livechat/static/src/legacy/public_livechat.js',
+            'im_livechat/static/src/legacy/models/website_livechat.js',
+            'im_livechat/static/tests/helpers/*.js',
```

Comment:
Should remove this, because already defined in `tests_assets` above. Due to definition in both assets, `helpers/mock_server` is deployed twice and generates following error on runbot: https://runbot.odoo.com/runbot/build/13918732